packages/fluent/scss/menu/_variables.scss

```diff
@@ -0,0 +1,315 @@
+/// @group menu
+$kendo-menu-popup-bg: var( --kendo-component-bg, initial ) !default;
+/// The border color of the menu popup.
+/// @group menu
+$kendo-menu-popup-border: initial !default;
+/// The shadow of the menu popup.
+/// @group menu
+$kendo-menu-popup-shadow: var( --kendo-box-shadow-depth-2, none ) !default;
+
+
+// Menu popup item
+
+/// Horizontal padding of the menu item in popup.
+/// @group menu
+$kendo-menu-popup-item-padding-x: map-get( $kendo-spacing, 2 ) !default;
+$kendo-menu-popup-item-padding-x-sm: map-get( $kendo-spacing, 2 ) !default;
+$kendo-menu-popup-item-padding-x-md: map-get( $kendo-spacing, 2 ) !default;
+$kendo-menu-popup-item-padding-x-lg: map-get( $kendo-spacing, 2 ) !default;
+
+/// Vertical padding of the menu item in popup.
+/// @group menu
+$kendo-menu-popup-item-padding-y: map-get( $kendo-spacing, 1 ) !default;
+$kendo-menu-popup-item-padding-y-sm: map-get( $kendo-spacing, 2 ) - map-get( $kendo-spacing, thin )!default;
+$kendo-menu-popup-item-padding-y-md: map-get( $kendo-spacing, 2 ) !default;
+$kendo-menu-popup-item-padding-y-lg: map-get( $kendo-spacing, 2 ) + map-get( $kendo-spacing, thin )!default;
+
+/// The end padding of the menu item in popup.
+/// @group menu
+$kendo-menu-popup-item-padding-end: calc( $kendo-menu-popup-item-padding-x * 2 + $kendo-icon-size ) !default;
+$kendo-menu-popup-item-padding-end-sm: calc( $kendo-menu-popup-item-padding-x-sm * 2 + $kendo-icon-size ) !default;
+$kendo-menu-popup-item-padding-end-md: calc( $kendo-menu-popup-item-padding-x-md * 2 + $kendo-icon-size ) !default;
+$kendo-menu-popup-item-padding-end-lg: calc( $kendo-menu-popup-item-padding-x-lg * 2 + $kendo-icon-size ) !default;
+
+/// The start margin of the menu item expand icon.
+/// @group menu
+$kendo-menu-popup-item-icon-margin-start-sm: map-get( $kendo-spacing, 0 ) !default;
+$kendo-menu-popup-item-icon-margin-start-md: map-get( $kendo-spacing, 0 ) !default;
+$kendo-menu-popup-item-icon-margin-start-lg: map-get( $kendo-spacing, 0 ) !default;
+
+/// The end margin of the menu item expand icon.
+/// @group menu
+$kendo-menu-popup-item-icon-margin-end-sm: calc( -1 * ($kendo-icon-size + $kendo-menu-popup-item-padding-x-sm) ) !default;
+$kendo-menu-popup-item-icon-margin-end-md: calc( -1 * ($kendo-icon-size + $kendo-menu-popup-item-padding-x-md) ) !default;
+$kendo-menu-popup-item-icon-margin-end-lg: calc( -1 * ($kendo-icon-size + $kendo-menu-popup-item-padding-x-lg) ) !default;
+
+/// The base shadow of focused menu item in popup.
+/// @group menu
+$kendo-menu-popup-focus-outline-offset: $kendo-menu-item-focus-outline-offset !default;
+$kendo-menu-popup-focus-outline-width: $kendo-menu-item-focus-outline-width !default;
+$kendo-menu-popup-focus-outline-style: $kendo-menu-item-focus-outline-style !default;
+
+/// The spacing between the menu items in popup.
+/// @group menu
+$kendo-menu-popup-item-spacing: map-get( $kendo-spacing, 0 ) !default;
+
+/// Sizes map for the menu.
+/// @group menu
+$kendo-menu-sizes: (
```

Comment:
These properties are applied to different HTML elements, and it will be a little mess if I make it in such way. For better readability I prefer to keep it with nested maps.